regression-test/framework/src/main/groovy/org/apache/doris/regression/suite/Suite.groovy

```diff
@@ -1330,8 +1330,8 @@ class Suite implements GroovyInterceptable {
         } while (timeoutTimestamp > System.currentTimeMillis() && (status == 'PENDING' || status == 'RUNNING'  || status == 'NULL' || status == 'CANCELED'))
         if (status != "SUCCESS") {
             logger.info("status is not success")
+            assertTrue(result.toString().contains("same table"))
```

Comment:
what does `same table` mean? what‘s the relation with method name `AllowCancel`

Reply:
The error message for completion is' Not allowed running Insert Overwrite on same table ', I have taken two words